### PR TITLE
feat(FOROME-1632): change a request for included/excluded variants

### PR DESCRIPTION
**Pull request by EvgeniySobolev**
[https://quantori.atlassian.net/browse/FOROME-1632](https://quantori.atlassian.net/browse/FOROME-1632)

### CODE REVIEW

File: src/store/dtree/step.store.ts [hunk 1]
```diff
@@ -15,6 +15,12 @@ export enum CreateEmptyStepPositions {
   FINAL = 'FINAL',
 }
 
+type IActiveStep = {
```
Comment (AndreyUstyumenko) [inline src/store/dtree/step.store.ts]: interface

File: src/pages/filter/dtree/components/modals/components/numeric-dialog/numeric-dialog.tsx [hunk 2]
```diff
@@ -60,6 +62,9 @@ export const NumericDialog = observer(
       modalsVisibilityStore.closeNumericDialog()
     }
 
+    const isOnDashboard: boolean =
```
Comment (AndreyUstyumenko) [inline src/pages/filter/dtree/components/modals/components/numeric-dialog/numeric-dialog.tsx]: no need to specify boolean

File: src/pages/filter/dtree/components/modals/components/inheritance-mode-dialog/inheritance-mode-dialog.tsx [hunk 3]
```diff
@@ -58,6 +60,9 @@ export const InheritanceModeDialog = observer(
       [attributeName, onAddFunc],
     )
 
+    const isOnDashboard: boolean =
```
Comment (AndreyUstyumenko) [inline src/pages/filter/dtree/components/modals/components/inheritance-mode-dialog/inheritance-mode-dialog.tsx]: no need to specify boolean

File: src/pages/filter/dtree/components/modals/components/gene-region-dialog/gene-region-dialog.tsx [hunk 4]
```diff
@@ -56,6 +58,9 @@ export const GeneRegionDialog = observer(
       [attributeName, onAddFunc],
     )
 
+    const isOnDashboard: boolean =
```
Comment (AndreyUstyumenko) [inline src/pages/filter/dtree/components/modals/components/gene-region-dialog/gene-region-dialog.tsx]: here too

File: src/pages/filter/dtree/components/modals/components/gene-region-dialog/gene-region-dialog.tsx [hunk 5]
```diff
@@ -56,6 +58,9 @@ export const GeneRegionDialog = observer(
...
+      dashboardStore.viewType === ViewTypeDashboard.Tile
```
Comment (AndreyUstyumenko) [inline src/pages/filter/dtree/components/modals/components/gene-region-dialog/gene-region-dialog.tsx]: maybe if it's on dtree it makes sense to create a property in dtree store

File: src/pages/filter/dtree/components/modals/components/enum-dialog/enum-dialog.tsx [hunk 6]
```diff
@@ -73,6 +74,9 @@ export const EnumDialog = observer(
     const listHeight =
       enumVariants.length > DEFAULT_COUNT ? 'calc(580px - 319px)' : 'auto'
 
+    const isOnDashboard: boolean =
+      dashboardStore.viewType === ViewTypeDashboard.Tile
```
Comment (AndreyUstyumenko) [inline src/pages/filter/dtree/components/modals/components/enum-dialog/enum-dialog.tsx]: here as well

File: src/pages/filter/dtree/components/modals/components/custom-inheritance-mode-dialog/custom-inheritance-mode-dialog.tsx [hunk 7]
```diff
@@ -58,6 +60,9 @@ export const CustomInheritanceModeDialog = observer(
       [attributeName, onAddFunc],
     )
 
+    const isOnDashboard: boolean =
```
Comment (AndreyUstyumenko) [inline src/pages/filter/dtree/components/modals/components/custom-inheritance-mode-dialog/custom-inheritance-mode-dialog.tsx]: and here

File: src/pages/filter/dtree/components/modals/components/compound-request-dialog/compound-request-dialog.tsx [hunk 8]
```diff
@@ -59,6 +61,9 @@ export const CompoundRequestDialog = observer(
       [attributeName, onAddFunc],
     )
 
+    const isOnDashboard: boolean =
```
Comment (AndreyUstyumenko) [inline src/pages/filter/dtree/components/modals/components/compound-request-dialog/compound-request-dialog.tsx]: and here

File: src/pages/filter/dtree/components/modals/components/compound-het-dialog/compound-het-dialog.tsx [hunk 9]
```diff
@@ -56,6 +58,9 @@ export const CompoundHetDialog = observer(
       [attributeName, onAddFunc],
     )
 
+    const isOnDashboard: boolean =
```
Comment (AndreyUstyumenko) [inline src/pages/filter/dtree/components/modals/components/compound-het-dialog/compound-het-dialog.tsx]: .

**Review by AndreyUstyumenko — APPROVED** [review]

**Review by QSdmitrioul — APPROVED** [review]

**Pull request merged**